src/UserManager.php

```diff
@@ -162,11 +166,15 @@ static function (array $user) use ($userIdClaim) {
         $users
       );
 
-    $this->logger->debug(json_encode($users, JSON_PRETTY_PRINT));
-
     $users = $this->userStorage->loadByProperties([$userIdField => $userIdsToKeep]);
     foreach ($users as $user) {
-      $this->logger->info($this->t('Retaining user @name.', ['@name' => $user->label()]));
+      $this->logger->info(
+        'Retaining user @name.',
+             [
+               '%name' => $user->label(),
+             ]
```

Comment:
For consistency with user info logging in `commitDeletionList`:

```suggestion
        'Retaining user %user (%id)',
        [
           '%user' => $user->label(),
           '%id' => $user->id(),
         ]
```